src/pages/landing/stocks/_headerSection.js

```diff
@@ -0,0 +1,396 @@
+import React from 'react'
+import styled from 'styled-components'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex } from 'components/containers'
+import StocksHeaderOverlayBg from 'images/svg/stocksHeaderOverlayBg.svg'
+import StocksHeaderOverlayBelowSectionBg from 'images/svg/stocksHeaderOverlayBelowSectionBg.svg'
+import StocksHeaderBelowSection from 'images/svg/stocksHeaderBelowSection.svg'
+import stocksHeaderBelowSectionM from 'images/svg/stocksHeaderBelowSection-m.svg'
```

Comment:
Can we create a separate folder for all these `svg` called `stock-indices`? This will reduce the number of files in the main `svg` folder.